src/store/products.js

```diff
@@ -78,30 +130,53 @@ export const increaseInventory = productName => {
   };
 };
 
-export default (state = products.initialState, action) => {
+export default (state = initialState, action) => {
   let { type, payload } = action;
   switch (type) {
-    case 'FILTER':
-      let productList = products.productList.filter(
-        item => item.category === payload
-      );
-      return { productList };
-    case 'REDUCE':
-      let newCount = products.productList.map((item => {
-        if (item.name === payload) {
-          item.inventoryCount--;
-          console.log(`${item.name} now has ${item.inventoryCount} in stock`);
-        }
-      }))
-      return { ...state, newCount };
-      case 'INCREASE':
-      let updatedCount = products.productList.map((item => {
-        if (item.name === payload) {
-          item.inventoryCount++;
-          console.log(`${item.name} now has ${item.inventoryCount} in stock`);
-        }
-      }))
-      return { ...state, updatedCount };
+    case 'GET':
+      return {
+        results: payload,
+      };
+    case 'DELETE':
+      console.log(`You just deleted:`, payload.title);
+      let newState = state.results.filter(item => item.id !== payload.id);
+      return { ...state, results: newState };
+    case 'POST':
+      console.log(`You just added:`, payload);
+      let addedItemState = state.results.push(payload);
+      return { ...state, results: addedItemState};
+
+
+      //return { ...state, results:payload};
+      //return { ...state, payload};
+      //return { ...state, payload};
+      // return {
+      //   ...state,
+      //   results: [...payload],
+      // };
+      //return {...state}
+
+    // case 'FILTER':
+    // let productList = productList.filter(
+    //   item => item.category === payload
+    // );
+    // return { productList };
+    // case 'REDUCE':
+    //   let newCount = productList.map((item => {
+    //     if (item.name === payload) {
+    //       item.inventoryCount--;
+    //       console.log(`${item.name} now has ${item.inventoryCount} in stock`);
+    //     }
+    //   }))
+    //   return { ...state, newCount };
+    //   case 'INCREASE':
+    //   let updatedCount = productList.map((item => {
+    //     if (item.name === payload) {
+    //       item.inventoryCount++;
+    //       console.log(`${item.name} now has ${item.inventoryCount} in stock`);
+    //     }
+    //   }))
+    // return { ...state, updatedCount };
     default:
       return state;
   }
```

Comment:
-1 point when I click on the category instead of filtering only those items it shows all the products, like if I click on men's clothing it shows me a list of all the products
- Make sure that your .env information is out of your .gitignore